docs/src/userguide/limited_api.rst

```diff
@@ -60,8 +60,8 @@ an external C library.
 
 Where the majority of the work involves interacting with Python objects the cost is likely to
 be more significant.  As an example, compiling the Cython compiler with the regular C API
-gives a ~35% speed-up compared to not compiling the Cython compiler.  Compiling the Cython
-compiler in the Limited API gives a 0-10% speed-up (depending on the exact version used).
+gives a ~40% speed-up compared to not compiling the Cython compiler.  Compiling the Cython
+compiler in the Limited API gives a 30-35% speed-up (depending on the exact version used).
```

Comment:
I wonder if this is still a good example for "significant cost". It looks quite close.

Reply:
I've changed "significant" it to "greater".

I can't easily think of a better example project, and it's only supposed to be rough estimate to help people know what to expect.